jazz_create-serverless-service/index.js

```diff
@@ -93,10 +92,6 @@ module.exports.handler = (event, context, cb) => {
                 var domain = (event.body.domain || "").toLowerCase();
                 var service_name = event.body.service_name.toLowerCase();
 
-                var bitbucketName = service_name;
```

Comment:
Call this function startServiceOnboarding

Reply:
In cooperated the change